docs/iam_roles.md

```diff
@@ -56,6 +56,16 @@ The additional permissions are:
   ]
 }
 ```
+## Permissions Boundaries
+Since v1.19.0, Kops supports AWS Permissions Boundaries. This enables you to use a policy (managed or custom) to set the maximum permissions that roles created by Kops will be able to grant to instances they're attached to. It can be useful to prevent possible privilege escalations.
```

Comment:
Should probably use kops_feature_table. See 485f95a88b2d739ed4459cc568c4c3e5f4b92e5c

Reply:
Sure! Just updated it.